packages/api/src/schema/db-schema.yaml

```diff
@@ -593,10 +593,27 @@ components:
           readOnly: true
           type: string
           example: webhookResponse
+        userId:
+          readOnly: true
+          type: string
+          index: true
         webhookId:
+          type: string
           index: true
         eventId:
+          type: string
+          index: true
+        event:
+          type: string
+          index: true
+        createdAt:
           index: true
+        response:
+          type: object
+          properties:
+            status:
+              type: number
+              index: true
```

Comment:
I couldn't tell if all these fields are in the `api-schema` as well, but I think at least `event` is not. Just to explain the idea between these 2 files:
 - `api-schema` gets generated into our public API docs, so it should contain the exact schema we want documented for our users
 - Then either for internal fields or custom properties we set on our fields (like `index:`), we set in this `db-schema` file.
   - Internal fields should have a `writeOnly` prop here (or custom code) so they get filtered on API responses
   - This gets deep merged on top of the API schema before we generate API code (since it gets deep merged, also no need to include duplicate props here like `type`)
 
So here, any field that is returned to the user (e.g. `event`) should also be in the `api-schema` so it shows up in our docs etc. Ideally with a `description` as well which will go in the doc

Does it make sense? For more info there's a little readme from when I did the schema split: https://github.com/livepeer/livepeer-com/blob/1c0773d9d6f5dfcffe0f149b19abb8dc4076bcd0/packages/api/src/schema/README.md#L1

Reply:
Ah yep thanks, will remember for next time!